app/views/interviews/reports/edit.html.erb

```diff
@@ -52,34 +64,34 @@
         </div>
 
         <%= f.input :start_time,
-          wrapper_html: { class: 'mb-0' },
-          input_html: {
-            class: 'datepicker-defaulting-to-now border-bkt-light-grey bkt-bg-light-grey3 rounded-5px height-5rem',
-          },
-          as: :date,
-          html5: true,
-          placeholder: 'Select date',
-          label: false
+                    wrapper_html: { class: 'mb-0' },
+                    input_html: {
+                      class: 'datepicker-defaulting-to-now border-bkt-light-grey bkt-bg-light-grey3 rounded-5px height-5rem',
+                    },
+                    as: :date,
+                    html5: true,
+                    placeholder: 'Select date',
+                    label: false
         %>
 
         <div class="fs-1_6rem mx-3">
           to
         </div>
 
         <%= f.input :end_time,
-          wrapper_html: { class: 'mb-0' },
-          input_html: {
-            class: 'datepicker-defaulting-to-end-of-year border-bkt-light-grey bkt-bg-light-grey3 rounded-5px height-5rem',
-          },
-          as: :date,
-          html5: true,
-          placeholder: 'Select date',
-          label: false
+                    wrapper_html: { class: 'mb-0' },
```

Comment:
@harold93 are those changes made by a linter? If so please create a PR implementing it to enforce consistency to the code base!